packages/web/.eslintrc

```diff
@@ -0,0 +1,52 @@
+{
+  "root": true,
+  "parser": "@typescript-eslint/parser",
+  "plugins": [
+    "@typescript-eslint"
+  ],
+  "extends": [
+    "eslint:recommended",
+    "airbnb",
+    "plugin:@typescript-eslint/eslint-recommended",
+    "plugin:@typescript-eslint/recommended"
+  ],
+  "settings": {
+    "import/extensions": [".js",".jsx",".ts",".tsx"],
+    "import/parsers": {
+      "@typescript-eslint/parser": [".ts",".tsx"]
+    },
+    "import/resolver": {
+      "node": {
+        "extensions": [".js",".jsx",".ts",".tsx"]
+      }
+    }
+  },
+  "rules": {
+    "no-unused-vars": "off",
```

Comment:
I'd suggest move to on

Reply:
If I understand correctly, the idea is that the rule is being replaced by following the `@typescript-eslint/no-unused-vars`, so they need to be disabled/enabled in pairs. Right, @SergeiKBB ?

Reply:
@dubbha Yes, you are right. If we turn on ` "no-unused-vars"` we will get a double error.
![image](https://user-images.githubusercontent.com/26768476/84817425-934c1b00-b01d-11ea-80f6-44c0089d451b.png)

Reply:
But as I understood I should remove it.

Reply:
Why? No, you shouldnt have. We don't need double errors, that's a default way to prepare an eslint config for typescript: you disable/enable these in pairs.

Reply:
Why? No, you shouldnt have. We don't need double errors, that's a default way to prepare an eslint config for typescript: you disable/enable these in pairs.